config/core/deployments/domainmapping-webhook.yaml

```diff
@@ -123,8 +132,10 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    role: domainmapping-webhook
```

Comment:
leave the `role` as is as the old selector requires it

Reply:
same as the one below

Reply:
added it back in